value-processor/src/org/immutables/value/processor/Immutables.generator

```diff
@@ -2479,6 +2497,25 @@ this.[n] = [valueFromValue v][invokeSuper v].[v.names.get]()[/valueFromValue];
 [/for]
 [/template]
 
+[template public buildFromMandatoryAttribute Invokable buildAttribute Attribute v String varName]
+[if v.isMandatory]
+if ([varName].[v.names.isSet]()) {
+  [buildAttribute v]
+}
+[else]
+  [buildAttribute v]
```

Comment:
is this extra indentation, please, check generated source so it have proper formatting for both with `if ({`  and without